sequencing/ATACSeq/jobSubmission/8_collateStage3QCMatrics.sh

```diff
@@ -84,7 +84,7 @@ fi
 ## Input in command line what set of peaks to perform analysis on: promoter peaks (prom) or all peaks (all)
 if [[ ! $3 == "PROM" ]] && [[ ! $3 == "ALL" ]];
 then
-  echo "No peak set specified. Please choose either PROM or ALL to select counts in promoter peaks or all peaks, respectively" ; exit 1; }     
+  { echo "No peak set specified. Please choose either PROM or ALL to select counts in promoter peaks or all peaks, respectively" ; exit 1; }     
```

Comment:
Good spot, the linter picked this up by I've stopped paying attention to it recently, my bad.